source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -516,6 +517,31 @@ message TransformationAddGlobalVariable {
 
 }
 
+message TransformationAddLocalVariable {
+
+  // Adds a local variable of the given type (which must be a pointer with
+  // Function storage class) to the given function, initialized to the given
+  // id.
+
+  // Fresh id for the local variable
+  uint32 fresh_id = 1;
+
+  // The type of the local variable
+  uint32 type_id = 2;
+
+  // The id of the function to which the local variable should be added
+  uint32 function_id = 3;
+
+  // Initial value of the variable
+  uint32 initializer_id = 4;
+
+  // True if and only if the value of the variable should be regarded, in
+  // general, as totally unknown and subject to change.  Stores to such
+  // variables can be performed in an arbitrary fashion.
+  bool value_is_arbitrary = 5;
```

Comment:
Could clarify that this is not like OpUndef (I think). It is more like: the fuzzer may add writes to this variable at any point.

Reply:
Done.